ssc/elections/2024H1/MARIUSZ_SABATH.md

```diff
@@ -0,0 +1,8 @@
+# NOMINEE NAME HERE
+Mariusz has been a maintainer on the helm charts project since it started and has been a long time contributer to SPIRE.
+
+Be sure to preserve the two empty spaces at the end of each line below.  
+**GitHub Handle:** @mrsabath  
+**Email Address:** mrsabath@gmail.com  
+**LinkedIn Profile:** https://www.linkedin.com/in/mariusz-sabath-b36b0b20/  
+**Current Affiliation:** Unknown  
```

Comment:
no known affiliation? or actually unknown? we could ask him :) @mrsabath?

Reply:
```suggestion
**Current Affiliation:** IBM Research 
```